releases/nightly-build/compose-files/source/Makefile

```diff
@@ -0,0 +1,128 @@
+# /*******************************************************************************
+#  * Copyright 2020 Intel
+#  *
+#  * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+#  * in compliance with the License. You may obtain a copy of the License at
+#  *
+#  * http://www.apache.org/licenses/LICENSE-2.0
+#  *
+#  * Unless required by applicable law or agreed to in writing, software distributed under the License
+#  * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  * or implied. See the License for the specific language governing permissions and limitations under
+#  * the License.
+#  *
+#  *******************************************************************************/
+
+.PHONY: help portainer portainer-down build build-ui run pull gen get-token ui-down down clean
+.SILENT: help get-token del-token
+
+include .env
+
+COMPOSE_FILES:=-f docker-compose-nexus-base.yml
+OPTIONS:=" arm64 no-secty no-ds mqtt dev ui " # Must have spaces around words for `filter-out` function to work properly
+
+ARGS:=$(wordlist 2,$(words $(MAKECMDGOALS)),$(MAKECMDGOALS))
+$(eval $(ARGS):;@:)
+
+# Order of processing these arguments is important so the overrides in the 'add` compose files occur properly.
+ifeq (dev, $(filter dev,$(ARGS)))
+	CORE_EDGEX_REPOSITORY=edgexfoundry
+	DEV:=-dev
+endif
+ifeq (arm64, $(filter arm64,$(ARGS)))
+	ARCH:=-arm64
+	KONG_UBUNTU:=-ubuntu
+endif
+ifneq (no-ds, $(filter no-ds,$(ARGS)))
+	COMPOSE_FILES:=$(COMPOSE_FILES) -f docker-compose-nexus-add-device-services.yml
+else
+	NO_DS:=-no-ds
+endif
+ifeq (mqtt, $(filter mqtt,$(ARGS)))
+	COMPOSE_FILES:=$(COMPOSE_FILES) -f docker-compose-nexus-add-mqtt.yml
+	MQTT:=-mqtt
+endif
+ifeq (no-secty, $(filter no-secty,$(ARGS)))
+	SECURITY:="NO"
```

Comment:
SECURITY:="NO" is not used and could be removed

Reply:
Cleaned up. THX!